### PR TITLE
Display information of the current repo after cloned/opened.

### LINKED ISSUE #20: Show information about a repository or workspace
**Is your feature request related to a problem? Please describe.**
Not related

**Describe the solution you'd like**
I would like to have information of where is the path of the repository Im targeting. 

**Describe alternatives you've considered**
- Display information of the current repo in the cli. 
- Display information of the workspace just right after cloned//opened. 

**Additional context**
currently there is no information of where the repo is cloned and if I want to store it on a different folder.

### PR DESCRIPTION
**Pull request by Kedap**
fix #20

### CODE REVIEW

**Comment by edfloreshz** [comment]
We could probably use something like [onefetch](https://github.com/o2sh/onefetch) to show me information.

Also, it might be better if we also showed this information using something like: `dm info` or `dm -i`

**Comment by Kedap** [comment]
But the issue says that it wants to show the information of the repository it is directed to, as well as information after the project has been cloned or opened.

File: src/utils/git_pull.rs [hunk 1]
```diff
@@ -182,3 +184,134 @@ fn fetch<'a>(
     let fetch_head = repo.find_reference("FETCH_HEAD")?;
     repo.reference_to_annotated_commit(&fetch_head)
 }
+
+pub fn status_short(path: String) -> Result<(), Error> {
+    let repo = Repository::open(&path)?;
+    if repo.is_bare() {
+        return Err(Error::from_str("cannot report status on bare repository"));
```
Comment (edfloreshz) [inline src/utils/git_pull.rs:191]:
```suggestion
        return Err(Error::from_str("Cannot report status on bare repository."));
```

File: src/utils/git_pull.rs [hunk 2]
```diff
@@ -182,3 +184,134 @@ fn fetch<'a>(
...
+    }
+    let statuses = repo.statuses(None)?;
+
+    let branch = get_branch(&repo);
+    println!(
+        "## {}",
```
Comment (edfloreshz) [inline src/utils/git_pull.rs:197]:
```suggestion
        "Branch: {}",
```

File: src/utils/git_pull.rs [hunk 3]
```diff
@@ -182,3 +184,134 @@ fn fetch<'a>(
...
+        branch.unwrap_or_else(|_| "HEAD (no branch)".to_string())
+    );
+
+    let modules = repo.submodules()?;
+    for sm in modules {
+        println!(
+            "# - submodule '{}' at {}",
```
Comment (edfloreshz) [inline src/utils/git_pull.rs:204]:
```suggestion
            "Submodule '{}' at {}",
```

**Pull request merged**